src/heap.cpp

```diff
@@ -12,19 +12,23 @@
 
 
 CoreAnalyzerHeapInterface* gCoreAnalyzerHeaps[HeapManagerLastOne];
+EnumHeapManager gCurrentHeap = HeapManagerPtMalloc;
+std::map<std::string, EnumHeapManager> gCAHeapers;
 
 
 void register_heap_managers() {
-	#ifndef _WIN32
-	// heap.cpp is used by ref.dll, so we need to only support switching heap in Linux
+	#ifndef WIN32
+	// heap.cpp is used by ref.dll, and we only support switching heap in Linux currently.
 	gCoreAnalyzerHeaps[HeapManagerPtMalloc] = get_pt_malloc_heap_manager();
 	gCoreAnalyzerHeaps[HeapManagerTcMalloc] = get_tc_malloc_heap_manager();
+	gCAHeapers.emplace(std::make_pair<std::string,EnumHeapManager>("pt", HeapManagerPtMalloc));
```

Comment:
The map is to consolidate all the changes when adding a new heap into a few places, for example, when adding a new heap, we don't need to change the `switch_heap` command.

Reply:
one question is should we combine the `gCAHeapers` and `gCoreAnalyzerHeaps`. They serve similar purpose.